src/pgduckdb_types.cpp

```diff
@@ -787,7 +789,7 @@ ConvertPostgresToDuckColumnType(Form_pg_attribute &attribute) {
 		break;
 	}
 	default: {
-		return duckdb::LogicalType::USER("UnsupportedPostgresType (Oid=" + std::to_string(type) + ")");
+		return duckdb::LogicalType::USER(duckdb::StringUtil::Format("UnsupportedPostgresType (Oid=%d)", type));
```

Comment:
`std::to_string` is inefficient:
- Underlyingly it calls `std::vsnprintf`,
- `std::sprintf` family internally invokes `std::localeconv_l` which acquires a mutex, thus hindering parallel execution

Reply:
Newer C++ standards (starting from C++26), STL implements `to_string` with `std::format`.

Some reference for the bad performance of `std::string`:
- https://en.cppreference.com/w/cpp/string/basic_string/to_string
- https://news.ycombinator.com/item?id=30472423